R/derive_var_extreme_flag.R

```diff
@@ -459,6 +459,35 @@ derive_worst_flag <- function(dataset,
 #'   mode = "last",
 #'   filter = AVISIT == "BASELINE" & DTYPE == "AVERAGE"
 #' )
+#'
+#' # OCCURDS Examples
+#' data("ae")
+#'
+#' # Most severe AE first occurrence per patient
+#' ae %>%
+#'   mutate(TEMP_AESEVN =
+#'          as.integer(factor(AESEV, levels = c("SEVERE","MODERATE","MILD")))) %>%
+#'   derive_var_extreme_flag(
+#'     new_var = AOCCIFL,
+#'     by_vars = vars(USUBJID),
+#'     order = vars(TEMP_AESEVN, AESTDY, AESEQ),
+#'     mode = "first"
+#'   ) %>%
```

Comment:
I would add
```
select(-TEMP_AESEVN) %>%
```
to make the derivation complete.

Reply:
@bundfussr as this is just an example to show how the function works is that really needed? i would agree with that for in the ADaM templates but i think its overkill in this respect as the temp variable is not displayed anyway.